diff -ur atlas-3.8.3.orig/CONFIG//include/atlconf.h atlas-3.8.3/CONFIG//include/atlconf.h
--- atlas-3.8.3.orig/CONFIG//include/atlconf.h	2010-04-08 15:48:00.000000000 +0000
+++ atlas-3.8.3/CONFIG//include/atlconf.h	2010-04-08 15:50:11.000000000 +0000
@@ -56,6 +56,13 @@
 #define MachIsHPPA(mach_) \
    ( 0 )
 #endif
+#ifdef __alpha__
+#define MachIsALPHA(mach_) \
+  ( __alpha__ )
+#else
+#define MachIsALPHA(mach_) \
+  ( 0 )
+#endif
 
 static char *f2c_namestr[5] = {"UNKNOWN","Add_", "Add__", "NoChange", "UpCase"};
 static char *f2c_intstr[5] =
Seulement dans atlas-3.8.3/CONFIG//include: atlconf.h~
diff -ur atlas-3.8.3.orig/CONFIG//src/probe_comp.c atlas-3.8.3/CONFIG//src/probe_comp.c
--- atlas-3.8.3.orig/CONFIG//src/probe_comp.c	2010-04-08 15:48:00.000000000 +0000
+++ atlas-3.8.3/CONFIG//src/probe_comp.c	2010-04-08 15:49:57.000000000 +0000
@@ -514,6 +514,8 @@
       return(sp);
    if (MachIsHPPA(arch))
      return(sp);
+   if (MachIsALPHA(arch))
+     return(sp);
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (!CompIsGcc(comp))
Seulement dans atlas-3.8.3/CONFIG//src: probe_comp.c~
diff -ur atlas-3.8.3.orig/CONFIG//src/SpewMakeInc.c atlas-3.8.3/CONFIG//src/SpewMakeInc.c
--- atlas-3.8.3.orig/CONFIG//src/SpewMakeInc.c	2010-04-08 15:48:00.000000000 +0000
+++ atlas-3.8.3/CONFIG//src/SpewMakeInc.c	2010-04-08 15:49:48.000000000 +0000
@@ -344,6 +344,8 @@
       return(sp);
    if (MachIsHPPA(arch))
      return(sp);
+   if (MachIsALPHA(arch))
+     return(sp);
    if (MachIsMIPS(arch))
       return((ptrbits == 64) ? "-mabi=64" : "-mabi=n32");
    if (!CompIsGcc(comp))
Seulement dans atlas-3.8.3/CONFIG//src: SpewMakeInc.c~
